### ISSUE TITLE
not working after upgrade to angular 9

### ISSUE THREAD

**Issue by mm580486** [fingerpich/jalali-angular-datepicker#121]
hi i have some problem with jalali pakage in angular 9 when i upgrade this pakage i got this error message 

```
ERROR in ./node_modules/ng2-jalali-date-picker/__ivy_ngcc__/ng2-jalali-date-picker.es5.js 1968:259-267
"export 'Renderer' was not found in '@angular/core'
ERROR in ./node_modules/ng2-jalali-date-picker/__ivy_ngcc__/ng2-jalali-date-picker.es5.js 2065:117-125
"export 'Renderer' was not found in '@angular/core'
ERROR in ./node_modules/ng2-jalali-date-picker/__ivy_ngcc__/ng2-jalali-date-picker.es5.js 2131:12-20
"export 'Renderer' was not found in '@angular/core'

    ERROR in node_modules/ngx-popover/PopoverContent.d.ts:1:81 - error TS2724: Module '"../@angular/core/core"' has no exported member 'Renderer'. Did you mean 'Renderer2'?

    1 import { AfterViewInit, ElementRef, ChangeDetectorRef, OnDestroy, EventEmitter, Renderer } from "@angular/core";
                                                                                      ~~~~~~~~

      node_modules/@angular/core/core.d.ts:5701:31
        5701 export declare abstract class Renderer2 {
                                           ~~~~~~~~~
        'Renderer2' is declared here.
    node_modules/ng2-jalali-date-picker/date-picker/date-picker.component.d.ts:16:100 - error TS2724: Module '"../../@angular/core/core"' has no exported member 'Renderer'. Did you mean 'Renderer2'?

    16 import { AfterViewInit, ChangeDetectorRef, ElementRef, EventEmitter, OnChanges, OnDestroy, OnInit, Renderer, SimpleChanges } from '@angular/core';
                                                                                                          ~~~~~~~~

      node_modules/@angular/core/core.d.ts:5701:31
        5701 export declare abstract class Renderer2 {
                                           ~~~~~~~~~
        'Renderer2' is declared here.
```
my pakages

```
{
  "name": "client",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build --prod --output-hashing=none",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^9.0.5",
    "@angular/common": "~9.0.5",
    "@angular/compiler": "~9.0.5",
    "@angular/core": "~9.0.5",
    "@angular/forms": "~9.0.5",
    "@angular/platform-browser": "~9.0.5",
    "@angular/platform-browser-dynamic": "~9.0.5",
    "@angular/router": "~9.0.5",
    "@ng-select/ng-select": "^3.7.0",
    "animate.css": "^3.7.2",
    "bootstrap": "^4.4.1",
    "core-js": "^3.5.0",
    "core-util-is": "^1.0.2",
    "font-awesome": "^4.7.0",
    "jalali-moment": "^3.3.3",
    "md5": "^2.2.1",
    "ng2-jalali-date-picker": "^2.2.7",
    "ng2-tooltip-directive": "^2.9.19",
    "ng2-validation": "^4.2.0",
    "ngx-chips": "2.1.0",
    "ngx-owl-carousel": "^2.0.7",
    "ngx-pagination": "^5.0.0",
    "ngx-popover": "0.0.16",
    "ngx-slick-carousel": "^0.4.6",
    "ngx-smart-modal": "^7.3.0",
    "ngx-smart-table": "^2.0.3",
    "ngx-toastr": "^11.2.1",
    "owl.carousel": "^2.3.4",
    "rxjs": "~6.5.3",
    "slick-carousel": "^1.8.1",
    "tslib": "^1.10.0",
    "zone.js": "~0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.900.5",
    "@angular/cli": "~9.0.5",
    "@angular/compiler-cli": "^9.0.5",
    "@angular/language-service": "~9.0.5",
    "@types/jasmine": "~3.5.0",
    "@types/jasminewd2": "^2.0.8",
    "@types/node": "^12.11.1",
    "codelyzer": "^5.1.2",
    "jasmine-core": "~3.5.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "^4.4.1",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage-istanbul-reporter": "^2.1.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.2",
    "protractor": "~5.4.0",
    "ts-node": "~8.5.4",
    "tslint": "~5.20.1",
    "typescript": "~3.7.5"
  }
}

```

**Comment by daliri72** [comment]
I have same problem

**Comment by gholami-mohammad** [comment]
same problem + 1

**Comment by daustany** [comment]
It has to replace Renderer with Renderer2 in __ivy_ngcc__, also the collaborator must merge the change with master, and update NPM ng2-jalali-datepicker package, but I don't know why the collaborator missed such issue !

**Comment by kamiyar-h** [comment]
I solved the problem for myself. hope help the others
replace Renderer to Renderer2 in:
1- node_modules/ng2-jalali-date-picker/__ivy_ngcc__/ng2-jalali-date-picker.js
lines: 1,1934
2- node_modules/ng2-jalali-date-picker/date-picker/date-picker.component.d.ts
lines: 16, 78

**Comment by mm580486** [comment]
Fixed #122
